packages/framework/src/Command/CreateDatabaseCommand.php

```diff
@@ -186,7 +186,7 @@ private function createCollationIfNotExists($collation, $locale)
             'SELECT 1
             FROM pg_collation
             WHERE collnamespace = (SELECT oid FROM pg_namespace WHERE nspname = \'pg_catalog\')
-            AND collencoding = pg_char_to_encoding(\'UTF8\')
+            AND (collencoding = pg_char_to_encoding(\'UTF8\') OR collencoding = -1)
```

Comment:
Sorry I don't understand this line, what does it mean?

Reply:
This creates a condition in which only encoding compatible with UTF-8 (first part of condition),or -1 if it works for any encoding

Reply:
https://www.postgresql.org/docs/10/catalog-pg-collation.html - 
collencoding - Encoding in which the collation is applicable, or -1 if it works for any encoding